gradle.properties

```diff
@@ -84,7 +84,7 @@ activationVersion=1.2.1
 
 annotationsVersion=15.0
 
-apacheTomcatVersion=8.5.51
+apacheTomcatVersion=9.0.34
```

Comment:
@labkey-susanh mentioned to check if we can move forward with the change to tomcat version here.

Reply:
Won't this make us incompatible with Tomcat 8.5.x? We haven't discussed dropping support for that version.

Reply:
I could try to change the version of embedded tomcat.

Reply:
Yes, embedding 9.0.x would be great. But we should continue to compile LabKey against 8.5.x.